src/new/wallet.js

```diff
@@ -1393,23 +1435,41 @@ class HathorWallet extends EventEmitter {
     return helpers.createTxFromData(ret.preparedData, this.getNetworkObject());
   }
 
+  /**
+   * @typedef BaseTransactionResponse
+   * @property {{hash:string,index:number,data:Buffer}[]} inputs
```

Comment:
The `BaseTransactionResponse` was created because the static code analyzers I'm using were not able to interpret the current jsdocs. Javascript code analysis is unable to fetch type data from Typescript, so I created this class as a temporary solution to improve this transition with basically the `Transaction` type while we don't migrate completely to TS.

From what I see now, there would be not much value in making this type definition very complete: it can be added upon when needed, by the functions that also update their JSdocs, as it has no interference on the code itself.